apps/condo/domains/organization/hooks/useEmployeeRolesTableData.tsx

```diff
@@ -246,7 +259,9 @@ export const useEmployeeRolesTableData: UseEmployeeRolesTableData = (connectedB2
                 },
             ],
         },
-    ].map(addNamesToPermissions(intl))
+    ]
+        .map(config => ({ ...config, permissions: config.permissions.filter(Boolean) }))
```

Comment:
what is this for?

Reply:
for filter `undefined` array elements in `permissions` (here if `hasPropertyDocumentsFeature` is false, then related `permissions` array will have two `undefined` elements)